### DISCUSSION TITLE
How to understand schadcn css color naming convention

### DISCUSSION THREAD

**Discussion by ImamJanjua** [shadcn-ui/ui#4216] [General]
Like the library has like its own naming convention for the colors but i dont get some parts of it because it seems very confusing. 
Like i went with color zink which is the standard I think so black and white. Now i get foreground and background. like foreground is black and background white. But why is than primary again black. 

Than we have primary-foreground which is like light gray. And secondary is a darker gray but why is than secondary-foreground again black. And than we have things like muted, accent which are like some other schades or gray and card and popover is again white and black

**Comment by geudrik** [comment]
https://ui.jln.dev/ Play with the theme builder. Will give you a good understanding of how the color names are used (and what they mean) in the context of shadcn
